File: CodeSnippets/javaSnippets/10Aug/cs10.java


class Core2web {

        public static void main(String[] args) {

		int var = 2;
                System.out.println(true == (13!=4) ? var+=4 : var*=4);
		/*Ternary operator (?) cannot be used with multilayer conditions like 'true == (13!=4)'*/
        }

}
/*
vivek@vivek-HIRAY:~/Core2Web-TREE-/CodeSnippets/javaSnippets/10Aug$ javac cs10.java 
cs10.java:8: error: unexpected type
                System.out.println(true == (13!=4) ? var+=4 : var*=4);
                                                   ^
  required: variable
  found:    value
1 error
 */

File: CodeSnippets/c codesnippets/10Aug/cs10.c


#include<stdio.h>

void main() {

        int i = 0;
        do {
                i++;

		if(i == 3)
			continue;

		printf("In Loop ");

        }while(i<3);

        printf("%d\n",i);

}
/*
vivek@vivek-HIRAY:~/Core2Web-TREE-/CodeSnippets/c codesnippets/10Aug$ cc cs10.c 
vivek@vivek-HIRAY:~/Core2Web-TREE-/CodeSnippets/c codesnippets/10Aug$ ./a.out 
In Loop In Loop 3
 */
